sites/_variables.scss

```
@@ -14,6 +14,11 @@ $search-gray: #e0e0e0;
 $profile-gray: #eaebed;
 $gray-light: #6f6f6f; // Light grey colour (Bootstrap override).
 
+// Brand variables
+$rule-width: 71px;
+$rule-thickness: .18em;
+$rule-color: $border-red;
```

Their technical SASS name can be adjusted accordingly too. I don't this following rule is only the default border width related to the first h1 and nothing else. And that width is only applicable to the border not to the h1 content.

```suggestion
$rule-sites-mainh1-border-width: 71px;
$rule-sites-mainh1-border-thickness: .18em;
$rule-sites-mainh1-border-color: $border-red;
```